config/templates/datacite.template.yml

```diff
@@ -1,6 +1,10 @@
 default: &default
+  rest_api: 'https://api.test.datacite.org'
   user: apitest
   password: apitest
+  prefix: '10.33555'
+  log_api: false
+  log_level: warn
```

Comment:
Maybe unused now that you have the nested `logger` config options?

Reply:
I didn't go over the template config file for what can be deleted, good catch.